table/block_based/block.h

```diff
@@ -559,13 +557,13 @@ class IndexBlockIter final : public BlockIter<IndexValue> {
     value_.clear();
   }
 
-  virtual void Prev() override;
+  void Prev() override;
 
-  virtual void Next() override;
+  void Next() override;
 
-  virtual void SeekToFirst() override;
+  void SeekToFirst() override;
 
-  virtual void SeekToLast() override;
+  void SeekToLast() override;
```

Comment:
Changes from this file are code cleaning up and are not related to the PR too? If possible, please move it to another PR.

Reply:
Thanks @siying for the suggestion, I have moved code cleaning to https://github.com/facebook/rocksdb/pull/6440.

Reply:
Done.